data/blog/dusty-daemon-receives-lts-grant.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Long-Term Support For Dusty Daemon"
+date: '2024-10-02'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['btcschellingpt', 'arvin', 'dergigi']
+images: ['/static/images/blog/57-dusty-daemon-lts-grant.jpg']
+summary: "OpenSats welcomes Dusty Daemon as an LTS Grantee."
+---
+
+OpenSats is excited to announce long-term support for the main driving force
+behind Splicing on the Lightning Network, [Dusty
+Daemon](https://github.com/ddustin). Dusty has contributed significantly to the
+Lightning Network [specifications](https://github.com/lightning/bolts/pull/863)
+and the design and implementation of Splicing.
+
+[Splicing](https://lightningsplice.com/splicing_explained.html) enhances the
+usability of the Lightning Network by allowing users to modify channel sizes
+instead of having to close and create new channels when size changes are
+required. This reduces the on-chain transactions required and improves the
+usability of the Lightning Network.
+
+With LTS support from OpenSats, Dusty will help extend the Splicing
+implementation across additional Lightning implementations and commence building
```

Comment:
Potentially re-word that part, as "Splicing implementation" followed by "Lightning implementations" sounds like a word repetition.

```suggestion
implementation across the Lightning ecosystem and commence building
```